packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -249,4 +253,55 @@ export class FrontendPluginImpl {
     Logger.info(Messages.EndDeploy(PluginInfo.DisplayName));
     return ok(undefined);
   }
+
+  public async generateArmTemplates(ctx: PluginContext): Promise<TeamsFxResult> {
+    Logger.info(Messages.StartGenerateArmTemplates(PluginInfo.DisplayName));
+
+    const bicepTemplateDir = path.join(
+      getTemplatesFolder(),
+      FrontendPathInfo.TemplateDir,
+      FrontendPathInfo.bicepTemplateFolderName
+    );
+
+    const moduleFilePath = path.join(bicepTemplateDir, FrontendPathInfo.moduleFileName);
+
+    const inputParameterOrchestrationFilePath = path.join(
+      bicepTemplateDir,
+      FrontendPathInfo.inputParameterOrchestrationFileName
+    );
+    const moduleOrchestrationFilePath = path.join(
+      bicepTemplateDir,
+      FrontendPathInfo.moduleOrchestrationFileName
+    );
+    const outputOrchestrationFilePath = path.join(
+      bicepTemplateDir,
+      FrontendPathInfo.outputOrchestrationFileName
+    );
+
+    const result: ScaffoldArmTemplateResult = {
+      Modules: {
+        simpleAuthProvision: {
+          Content: await fs.readFile(moduleFilePath, "utf-8"),
```

Comment:
I guess there is a constant for "utf-8", if not, please help create one.

Reply:
Same as above one.

Reply:
Zhijie will help to make it a constant string in another PR.